packages/vscode-extension/src/qm/vsc_ui.ts

```diff
@@ -446,7 +448,85 @@ export class VsCodeUI implements UserInteraction {
   }
 
   async selectFolder(config: SelectFolderConfig): Promise<Result<SelectFolderResult, FxError>> {
-    return this.selectFileInQuickPick(config, "folder", config.default);
+    if (!TreatmentVariableValue.useFolderSelection) {
+      // control group
+      return this.selectFileInQuickPick(config, "folder", config.default);
```

Comment:
Which folder will be shown since the default folder might not exist?

Reply:
It depends on the behavior of VS Code. Currently it's user's home folder.